DevCycle.SDK.Server.Cloud/Api/DVCCloudClient.cs

```diff
@@ -182,7 +182,7 @@ public async Task<DVCResponse> TrackAsync(User user, Event userEvent)
             }
         }
 
-        public override void Dispose()
+        public override async ValueTask DisposeAsync()
```

Comment:
seems like this was added in C# 8. Does using this increase our minimum supported language version?